dnsseed.go

```diff
@@ -114,44 +116,50 @@ func creep() {
 			go func(addr *wire.NetAddress) {
 				defer wgCreep.Done()
 
-				host := net.JoinHostPort(addr.IP.String(), strconv.Itoa(int(addr.Port)))
-				p, err := peer.NewOutboundPeer(&cfg, host)
+				err := pollPeer(cfg, addr, onAddr)
 				if err != nil {
-					log.Warnf("NewOutboundPeer on %v: %v",
-						host, err)
-					return
-				}
-				amgr.Attempt(addr.IP)
-				conn, err := net.DialTimeout("tcp", p.Addr(), nodeTimeout)
-				if err != nil {
-					log.Warnf("DialTimeout on %v: %v", host, err)
-					return
-				}
-				err = p.AssociateConnection(conn)
-				if err != nil {
-					log.Warnf("AssociateConnection on %v: %v", host, err)
-					p.Disconnect()
-					return
+					log.Warnf(err.Error())
+					if defaultSeeder != nil && addr == defaultSeeder {
+						panics.Exit(log, "failed to poll default seeder")
```

Comment:
panics.Exit also prints stack trace and everything.
Do we really want a stack trace here?